publications/views.py

```diff
@@ -168,4 +165,63 @@ def change_useremail(request):
         send_mail(subject, message, from_email= settings.EMAIL_HOST_USER,recipient_list=[email_new])
         logout(request)
 
-    return  render(request,'changeuser.html')
\ No newline at end of file
+    return  render(request,'changeuser.html')
+
+def cuser(request):
```

Comment:
This function name is not "speaking" - please do not save characters here but rename the function to the much more easily understandable `change_user`.

Reply:
it is a redundant function .it is no longer in use and is redacted.